repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/ActiveMQPipelineRunner.java

```diff
@@ -0,0 +1,11 @@
+package fr.inria.spirals.repairnator.realtime;
+
+import fr.inria.spirals.repairnator.dockerpool.AbstractPoolManager;
+
+public class ActiveMQPipelineRunner extends AbstractPoolManager {
+
+
+    public ActiveMQPipelineRunner(RTScanner rtScanner) {
+    }
```

Comment:
add comment explaining that the implementation will come in a future pull-request

Reply:
Fixed :) .

Reply:
I don't see the change. Push it on this PR?

Reply:
There are two new commits down below :). The first one address this.